tools/python/checkupstreamversion.py

```diff
@@ -0,0 +1,37 @@
+import json
```

Comment:
Could we get a `#!/usr/bin/env python` intro line and add the execute bit please?